content/_code-samples/send-xrp/py/send-xrp.py

```diff
@@ -1,6 +1,6 @@
 # Example Credentials ----------------------------------------------------------
 from xrpl.wallet import Wallet
-test_wallet = Wallet(seed="sn3nxiW7v8KXzPzAqzyHXbSSKNuN9", sequence=16237283)
+test_wallet = Wallet.from_seed("sn3nxiW7v8KXzPzAqzyHXbSSKNuN9")
 print(test_wallet.classic_address) # "rMCcNuTcajgw7YTgBy1sys3b89QqjUrMpH"
```

Comment:
I suggest using the new shorter alias here.

```suggestion
print(test_wallet.address) # "rMCcNuTcajgw7YTgBy1sys3b89QqjUrMpH"
```